frontend/src/components/User/Login.jsx

```diff
@@ -46,12 +53,36 @@ export default function LoginPage() {
       .then(() => setStatus("complete"));
   };
 
+  const doGithubLogin = (code) => {
+    setStatus("loading");
+    sendRequest("/github", { code }).then(
+      (user) => {
+        setUserProfile(user);
+        setStatus("githubAuth");
+      },
+      (err) => {
+        setComplete({
+          rstatus: "error",
+          msg: err.message,
+          buttonFn: () => {
+            setStatus("githubAuth");
```

Comment:
Currently, this button lead the user to the `GithubLoginFinish` state, and then the dashboard page with no user logged in.
This button should redirect to the login state when user fail to login with github.